requiremenets.txt

```diff
@@ -0,0 +1,10 @@
+Django==1.11.13
```

Comment:
What is requiremenets.txt about ? Why do we have two requirements.txt files, and one of them is named wrongly ?